packages/angular/mf/mf.ts

```diff
@@ -58,9 +58,14 @@ async function loadRemoteContainer(remoteName: string) {
     ? remoteUrlDefinitions[remoteName]
     : await resolveRemoteUrl(remoteName);
 
-  const containerUrl = `${remoteUrl}${
-    remoteUrl.endsWith('/') ? '' : '/'
-  }remoteEntry.mjs`;
+  const remoteEntryFileName =
+    remoteUrl.endsWith('.mjs') || remoteUrl.endsWith('.js')
+      ? remoteUrl.split('/').pop()
+      : undefined;
+
+  const containerUrl = `${remoteUrl}${remoteUrl.endsWith('/') ? '' : '/'}${
+    remoteEntryFileName ?? 'remoteEntry.mjs'
+  }`;
```

Comment:
We could simplify it and only process and build the URL if it's not a fully-fledged URL already:
```suggestion
  let containerUrl = remoteUrl;
  if (!remoteUrl.endsWith('.mjs') && !remoteUrl.endsWith('.js')) {
    containerUrl = `${remoteUrl}${
      remoteUrl.endsWith('/') ? '' : '/'
    }remoteEntry.mjs`;
  }
```